android/jni/Android.mk

```diff
@@ -4,11 +4,45 @@ SRC := ../..
 include $(CLEAR_VARS)
 include $(LOCAL_PATH)/Locals.mk
 
+LOCAL_CFLAGS += -D_7ZIP_ST -D__SWITCH__
```

Comment:
`__SWITCH__`?

-[Unknown]

Reply:
It only makes sense if you go look at the c file in question, CpuArch in lzma. It avoids using the getauxflags function on Switch and Vita to check for NEON, and that workaround for the lack of the function works fine for us.

Reply:
Heh, a bit gross, but got it.  We don't get getauxval on some Android archs because we're still supporting older targets iirc.  cpu_features had this issue too but I think had a cleaner check.

-[Unknown]